src/gen8_post_processing.c

```diff
@@ -328,6 +328,28 @@ static const uint32_t pp_yuv420p8_scaling_gen8[][4] = {
 #include "shaders/post_processing/gen8/conv_nv12.g8b"
 };
 
+static const uint32_t pp_8bit_420_rgb32_scaling_gen8[][4] = {
+#include "shaders/post_processing/gen8/conv_8bit_420_rgb32.g8b"
+};
+
+struct i965_kernel pp_common_scaling_gen8[] = {
+    {
+        "8bit to 8bit",
+        0,
+        pp_yuv420p8_scaling_gen8,
+        sizeof(pp_yuv420p8_scaling_gen8),
+        NULL,
+    },
+
+    {
+        "8bit 420 to rgb32",
+        1,
```

Comment:
how about 10bit 420 to rgb32?

Reply:
the legacy path is still used. Of course we can add support for 10bit 420 to rgb32 in the future.